src/xdocs/cmdline.xml.vm

```diff
@@ -96,26 +96,27 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-s</code> option. Default value is 8.
         </li>
         <li>
-          <code>-t, --tree</code> - Prints Abstract Syntax Tree(AST) of the checked file. The option
-          cannot be used other options and requires exactly one file to run on to be specified.
+          <code>-t, --tree</code> - This option is used to display the Abstract Syntax Tree (AST)
```

Comment:
Are comments completely gone in this mode?

It may be beneficial to mention that. Ex: `used to display the Abstract Syntax Tree (AST)
Comment of the specified file` => `used to display only the Abstract Syntax Tree (AST)
Comment of the specified file without any comments`

Reply:
> Are comments completely gone in this mode?


```
E:\MANISH\Workspace>java -jar checkstyle-10.13.0-all.jar -t Test.java
COMPILATION_UNIT -> COMPILATION_UNIT [7:0]
`--CLASS_DEF -> CLASS_DEF [7:0]
    |--MODIFIERS -> MODIFIERS [7:0]
    |--LITERAL_CLASS -> class [7:0]
    |--IDENT -> Test [7:6]
    `--OBJBLOCK -> OBJBLOCK [7:11]
        |--LCURLY -> { [7:11]
        `--RCURLY -> } [14:0]

E:\MANISH\Workspace>
```
yep, no comment nodes (source code used is same as above post)
@rnveach , Will add your suggestions all at once after confirmation on above two posts

Reply:
Let me know if you still need anything from me and what specifically.

Reply:
Done..

> Let me know when documentation in this PR is updated.

I've updated docs according to your suggestions.. 

> Let me know if you still need anything from me and what specifically.

Its a bit confusing to find targeted comments in this PR now 😅 
@rnveach , new issue created at #14780 for discussion. Please confirm if I've mentioned everything there